src/test/java/uk/gov/digital/ho/proving/income/api/FinancialStatusServiceTest.java

```diff
@@ -69,15 +69,15 @@ public void shouldReturnIndividualsInCorrectOrder() {
         when(mockIncomeValidationService.validate(any())).thenReturn(getCategoryChecks());
 
 
-        FinancialStatusCheckResponse response = financialStatusService.calculateResponse(LocalDate.now(), 0, getIncomeRecordsOutOfOrder());
+        FinancialStatusCheckResponse response = financialStatusService.calculateResponse(LocalDate.now(), 0, getIncomeRecords());
 
 
         assertThat(response.individuals().size()).isEqualTo(2).withFailMessage("The correct number of individuals should be returned");
-        assertThat(response.individuals().get(0).nino()).isEqualTo("A").withFailMessage("The applicant should be returned first");
-        assertThat(response.individuals().get(1).nino()).isEqualTo("B").withFailMessage("The partner should be returned second");
+        assertThat(response.individuals().get(0).nino()).withFailMessage("The applicant should be returned first").isEqualTo("A");
```

Comment:
I was just wondering why you put the test last?  I don't think it reads as well..

Reply:
I know, I agree it's less readable but it won't pick up any fail messages unless they are before the assertion.

Would like breaks make it clearer?
```
assertThat(response.individuals().get(0).nino())
    .withFailMessage("The applicant should be returned first")
    .isEqualTo("A");
```

Reply:
Well that's a bit poor isn't it?  OK, given that you have to register the fail message first I'll leave the line formatting to your judgement.